test/support/builders/metadata/base.js

```diff
@@ -111,6 +112,9 @@ module.exports = class BaseMetadataBuilder {
   }
 
   ino (ino /*: number */) /*: this */ {
+    if (process.platform === 'win32') {
+      this.doc.fileid = statsBuilder.fileIdFromNumber(ino)
```

Comment:
Should we use the `StatsBuilder` when calling the `stats()` method as well?

Reply:
Probably :)

Reply:
OK. It's a confusing name then. But it's out of scope.